charts/pega/values-minimal.yaml

```diff
@@ -44,6 +44,20 @@ global:
     dataSchema: "YOUR_DATA_SCHEMA"
     customerDataSchema: ""
 
+  customArtifactory:
```

Comment:
same comment as for other yaml>
customArtifactory:
If you use a secured custom artifactory to manager your JDBC driver, provide the authentication details below by filling in the appropriate authentication section, either basic or apiKey.

authentication:

Provide the basic authentication credentials or the API key authentication details to satisfy your custom artifactory authentication mechanism.

basic:
username: "YOUR_ARTIFACTORY_USERNAME"
password: "YOUR_ARTIFACTORY_PASSWORD"
apiKey:
headerName: "YOUR_ARTIFACTORY_DEDICATED_APIKEY_HEADER"
value: "YOUR_ARTIFACTORY_API_KEY"

Leave customArtifactory.enableSSLVerification enabled to ensure secure access to your custom artifactory; when customArtifactory.enableSSLVerification is false, SSL verification is skipped and establishes an insecure connection.

enableSSLVerification: true

Provide a required domain certificate for your custom artifactory server; if none is required, leave this field blank.
certificate: